cms/templates/base.html

```diff
@@ -72,9 +72,11 @@
       <%block name="page_alert"></%block>
       </div>
 
-      <div id="content" tabindex="-1">
-      <%block name="content"></%block>
-      </div>
+      <main id="main" aria-label="content" tabindex="-1">
```

Comment:
To be consistent, can we uppercase aria-label="Content" everywhere it appears in this PR.  All the other landmark regions have uppercase labels.